From 7dd11ef07549950f5c42d1561f3681a98aa7ca61 Mon Sep 17 00:00:00 2001
From: Luo Jie <jie.luo@oss.qualcomm.com>
Date: Wed, 3 Sep 2025 14:07:19 +0800
Subject: [PATCH 506/512] arm64: dts: qcom: Update PCS nodes for
 ipq9570-kiwi-dvk board

PCS node names are updated as same as the upstream version.

Signed-off-by: Luo Jie <jie.luo@oss.qualcomm.com>
---
 arch/arm64/boot/dts/qcom/ipq9570-kiwi-dvk.dts | 42 +++++++++++++++++--
 1 file changed, 39 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/ipq9570-kiwi-dvk.dts b/arch/arm64/boot/dts/qcom/ipq9570-kiwi-dvk.dts
index 24852947da01..bcee30fd74a3 100644
--- a/arch/arm64/boot/dts/qcom/ipq9570-kiwi-dvk.dts
+++ b/arch/arm64/boot/dts/qcom/ipq9570-kiwi-dvk.dts
@@ -74,6 +74,42 @@
 	status = "okay";
 };
 
+&pcs0 {
+        status = "okay";
+};
+
+&pcs0_ch0 {
+        clocks = <&nsscc NSS_CC_UNIPHY_PORT1_RX_CLK>,
+                 <&nsscc NSS_CC_UNIPHY_PORT1_TX_CLK>;
+        clock-names = "rx",
+                      "tx";
+        status = "okay";
+};
+
+&pcs1 {
+        status = "okay";
+};
+
+&pcs1_ch0 {
+        clocks = <&nsscc NSS_CC_UNIPHY_PORT5_RX_CLK>,
+                 <&nsscc NSS_CC_UNIPHY_PORT5_TX_CLK>;
+        clock-names = "rx",
+                      "tx";
+        status = "okay";
+};
+
+&pcs2{
+        status = "okay";
+};
+
+&pcs2_ch0 {
+        clocks = <&nsscc NSS_CC_UNIPHY_PORT6_RX_CLK>,
+                 <&nsscc NSS_CC_UNIPHY_PORT6_TX_CLK>;
+        clock-names = "rx",
+                      "tx";
+        status = "okay";
+};
+
 &qcom_ppe {
 	ethernet-ports {
 		#address-cells = <1>;
@@ -85,7 +121,7 @@
 			managed = "in-band-status";
 			label = "wan";
 			sfp = <&sfp0>;
-			pcs-handle = <&pcsuniphy0_ch0>;
+			pcs-handle = <&pcs0_ch0>;
 			local-mac-address = [00 00 00 00 00 00];
 			clocks = <&nsscc NSS_CC_PORT1_MAC_CLK>,
 				 <&nsscc NSS_CC_PORT1_RX_CLK>,
@@ -106,7 +142,7 @@
 			phy-mode = "2500base-x";
 			label = "lan1";
 			phy-handle = <&phy0>;
-			pcs-handle = <&pcsuniphy1_ch0>;
+			pcs-handle = <&pcs1_ch0>;
 			local-mac-address = [00 00 00 00 00 00];
 			clocks = <&nsscc NSS_CC_PORT5_MAC_CLK>,
 				 <&nsscc NSS_CC_PORT5_RX_CLK>,
@@ -128,7 +164,7 @@
 			managed = "in-band-status";
 			label = "lan2";
 			phy-handle = <&phy1>;
-			pcs-handle = <&pcsuniphy2_ch0>;
+			pcs-handle = <&pcs2_ch0>;
 			local-mac-address = [00 00 00 00 00 00];
 			clocks = <&nsscc NSS_CC_PORT6_MAC_CLK>,
 				 <&nsscc NSS_CC_PORT6_RX_CLK>,
-- 
2.34.1

